server/src/main/java/org/apache/iotdb/db/qp/physical/sys/AuthorPlan.java

```diff
@@ -39,12 +39,13 @@
 public class AuthorPlan extends PhysicalPlan {
 
   private AuthorOperator.AuthorType authorType;
-  private String userName;
   private String roleName;
   private String password;
   private String newPassword;
   private Set<Integer> permissions;
   private PartialPath nodeName;
+  private String userName;
+  private String loginUserName;
```

Comment:
I think maybe we can push loginUserName down to PhysicalPlan. It is often meaningful to know who issued this plan.

Reply:
done. thank you! 
index is for rowRocord, it was there before my edit.